### ISSUE TITLE
Password reset

### ISSUE THREAD

**Issue by Hazy1369** [sologenic/sologenic-expo-wallet#94]
Can't send out tokens bc my password isn't working. How to reset?

**Comment by Kenakpe** [comment]
Did you get help?

**Comment by Hazy1369** [comment]
I wrote support sunday with no response so far

Sent from Yahoo Mail on Android 
 
  On Fri, Nov 19, 2021 at 6:45 AM, ***@***.***> wrote:   


Did you get help?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by DeltaLeet00** [comment]
Same, been asking for assistance and no response

**Comment by Hazy1369** [comment]
First the password quit working. Then the wallet completely locked up. Is there a scam happening here? Now it's been over a week an no response from support.

**Comment by DeltaLeet00** [comment]
I dont but im getting upset, i transferred xrp to get some solo and didnt think about it. Decided to transfer the rest and got on the app to get more solo and it starts asking for a password...i find it very frustrating that i cant just use my pin or yhe fact that there ia no reset password for the app...and i have tried in vein to contact someone...they can post of the offical sologenic twitter but cant respond to messages apparently....

**Comment by Hazy1369** [comment]
I had almost $400 in the wallet & now consider stolen

**Comment by Hazy1369** [comment]
no help at all

**Comment by DeltaLeet00** [comment]
I would have to agree with you there, i feel lile if i cant have acess to my coin then you are Basically stealing it from me due to the lack of support

**Comment by Kenakpe** [comment]
My over $1500 is basically gone I have my secret phrase but for some
reason, the wallet password is even more important than the secret phrase.
No help whatsoever.
Ken Akpe
K & S Consulting.

**Comment by martindec2000** [comment]
Hi there.. same situation here.. pls tell me if u get any help...

**Comment by Hazy1369** [comment]
Yes, If anyone finds help with this issue please share with all.
Mark Hazlewood 

    On Friday, November 26, 2021, 11:49:01 AM EST, martindec2000 ***@***.***> wrote:  
 
 


Hi there.. same situation here.. pls tell me if u get any help...

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by DeltaLeet00** [comment]
Yea unfortunately i have tried to reach out multiple ways with no help, all i can do is look at my account..I dont understand how they can keep posting on social media without looking at any communications... they are happy to push their platform and what progress they are making but refuse to reapond to requests for help. Starting to smell like a scam to me..
